src/core/textures/VideoBaseTexture.js

```diff
@@ -54,13 +55,8 @@ export default class VideoBaseTexture extends BaseTexture
         this.width = source.videoWidth;
         this.height = source.videoHeight;
 
-        /**
-         * Should the base texture automatically update itself, set to true by default
-         *
-         * @member {boolean}
-         * @default true
-         */
-        this.autoUpdate = false;
+        this._autoUpdate = true;
+        this._isAutoUpdating = false;
```

Comment:
Do you need both flags here? I feel like you could get away with just using `_autoUpdate`.